oaknational/Oak-Web-Application
# Feat/pupil 774/fix glipse tier examboard programmes

## Description

Music year: 1951

- List of changes

There is no longer a glitch when you go back to a tier options page for a subject with examboard options

I have achieved this by creating a page specifically to be landed on with a default examboard which means the state change to select tier is handled statically rather then client side therefore theres no glitch



## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2528--oak-web-application.netlify.thenational.academy/pupils/years/year-10/subjects
2. Click on a subject with tiers and examboard, e.g. biology, then click through to the unit page
3. You should see if you click back you are taken ti the tier selection page, without visible glitch, but the url is slightly different to prviously

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
